lib/demos/reference/motion_demo_shared_x_axis_transition.dart

```diff
@@ -89,17 +89,13 @@ class _CoursePage extends StatelessWidget {
 
     return ListView(
       children: [
-        const Padding(
-          padding: EdgeInsets.symmetric(vertical: 8),
-        ),
+        const SizedBox(height:16),
         Text(
           localizations.demoSharedXAxisCoursePageTitle,
           style: Theme.of(context).textTheme.headline5,
           textAlign: TextAlign.center,
         ),
-        const Padding(
-          padding: EdgeInsets.symmetric(vertical: 5),
-        ),
+        const SizedBox(height:16),
```

Comment:
Oops, I didn't see that this was originall 5, so the height should be 10 (and there should be a space after the :